src/control/server/main.go

```diff
@@ -88,6 +89,14 @@ func serverMain() error {
 	if f != nil {
 		defer f.Close()
 	}
+	// Load up certificates
+	creds, err := security.LoadServerCreds(config.CaCert,
+		config.Cert,
```

Comment:
style nit: Do these really each need to be on their own line?

Reply:
He probably hit the 80 char limit so reformatted it.

Reply:
Yea its only like that because it will hit the 80 character limit. I guess they don't all have to be on their own line it just looked a bit cleaner that way.